gbm_drm_gen/drmgen.py

```diff
@@ -101,27 +101,28 @@ def to_3ML_response(self, ra, dec):
 
         return response
 
-    def to_fits(self,ra,dec,filename,overwrite):
+    def to_3ML_response_direct_sat_coord(self, ra, dec):
```

Comment:
I think the function should be better named. Or, maybe the function arguments of the original function extended.